pkgs/development/tools/misc/iozone/default.nix

```diff
@@ -28,15 +28,33 @@ stdenv.mkDerivation rec {
 
   buildFlags = target;
 
+  buildDeps = [ gnuplot ];
+
+  enableParallelBuilding = true;
+
   installPhase = ''
-    mkdir -p $out/{bin,share/doc,share/man/man1}
+    mkdir -p $out/{bin,share/doc,libexec,share/man/man1}
     install docs/iozone.1 $out/share/man/man1/
     install docs/Iozone_ps.gz $out/share/doc/
     install -s src/current/{iozone,fileop,pit_server} $out/bin/
+    install src/current/gengnuplot.sh $out/libexec/
+    # The graphing scripts need some fixups.
+    echo '#!/bin/sh' > $out/libexec/Generate_Graphs
```

Comment:
Can't help myself but suggest that this is more succinctly stated as `sed "1i#! $shell" src/current/Generate_Graphs >$out/libexec/Generate_Graphs`.